test/extended/project/project.go

```diff
@@ -136,13 +140,18 @@ var _ = g.Describe("[Feature:ProjectAPI] ", func() {
 			o.Expect(err).NotTo(o.HaveOccurred())
 			select {
 			case event := <-fromNowWatch.ResultChan():
-				g.Fail(fmt.Sprintf("unexpected event %s %#v", event.Type, event.Object))
+				// there might be a race when the ginkgo is fast and start deleting the project and this
```

Comment:
not sure how gingko can start deleting project before this blocking code ends. does this run in serial suit? it should